{"trial_id": "1", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 32, "max_value": 128, "step": 16, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 16, "max_value": 128, "step": 16, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "learning_rate", "default": 0.001, "conditions": [], "values": [0.001, 0.01, 0.1], "ordered": true}}], "values": {"units_1": 48, "units_2": 32, "learning_rate": 0.001}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\base_tuner.py\", line 270, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\base_tuner.py\", line 261, in _run_and_update_trial\n    tuner_utils.convert_to_metrics_dict(\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\tuner_utils.py\", line 225, in convert_to_metrics_dict\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\tuner_utils.py\", line 225, in <listcomp>\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\tuner_utils.py\", line 238, in convert_to_metrics_dict\n    best_value, _ = _get_best_value_and_best_epoch_from_history(\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\tuner_utils.py\", line 209, in _get_best_value_and_best_epoch_from_history\n    objective_value = objective.get_value(metrics)\n  File \"c:\\ANACONDA3\\ENVS\\MLOPS-TFX\\lib\\site-packages\\keras_tuner\\engine\\objective.py\", line 57, in get_value\n    return logs[self.name]\nKeyError: 'val_binary_accuracy'\n"}